x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.tsx

```diff
@@ -57,18 +57,21 @@ const AlertSummaryViewComponent: React.FC<{
   eventId: string;
   isDraggable?: boolean;
   timelineId: string;
-  title?: string;
-}> = ({ browserFields, data, eventId, isDraggable, timelineId, title }) => {
+  title: string;
+  goToTable: () => void;
+}> = ({ browserFields, data, eventId, isDraggable, timelineId, title, goToTable }) => {
   const summaryRows = useMemo(
     () => getSummaryRows({ browserFields, data, eventId, isDraggable, timelineId }),
     [browserFields, data, eventId, isDraggable, timelineId]
   );
 
   return (
-    <>
-      <EuiSpacer size="s" />
```

Comment:
💅 I moved this spacer to the parent so it has full control over the layout and we can get rid of the `ReactFragment`

Reply:
nice refactor @janmonschke! 🚀